staging/src/k8s.io/kubelet/config/v1beta1/types.go

```diff
@@ -90,21 +90,21 @@ type KubeletConfiguration struct {
 
 	// enableServer enables Kubelet's secured server.
 	// Note: Kubelet's insecure port is controlled by the readOnlyPort option.
-	// Dynamic Kubelet Config (beta): If dynamically updating this field, consider that
+	// Dynamic Kubelet Config (deprecated): If dynamically updating this field, consider that
```

Comment:
I'm not sure I see the reason for editing something in a beta api like this?

Reply:
the `(beta)` here indicates the status of the `DynamicKubeletConfig` feature. I suggest to highlight the fact that the feature will be gone soon. Alternatively, I can change `(deprecated)` to something like `to be removed`?

Reply:
Presumably in v1beta1 it was beta, though? I don't know if I object, this just seems like modifying the past...

Reply:
I understand. It is a valid point. However, the problem is that when feature will be removed, this API behavior will also change retroactively. This is why I think it makes sense to update the description now.

Reply:
OK

Reply:
I found the `Dynamic Kubelet Config (deprecated)` comments here confusing, because for a moment I thought it implied the `KubeletConfig` was being deprecated.

@SergeyKanzhelev I would suggest that we _remove_ all mention of DynamicKubeletConfig from the KubeletConfig comments here instead. Since it's being deprecated, IMO this advice is no longer relevant, and since KubeletConfig is a beta API I think we can change this.

Reply:
I think it's good to document the current state even if that is confusing. I agree it could be worded better, e.g. "If DynamicKubeletConfig (deprecated; default off) is on, then ..."

I don't feel too strongly. The pre-existing sentences are not super clear, either.

Reply:
I can send the proposed clarified text as a separate PR.